### DISCUSSION TITLE
Does the server support RTP Extension Headers? RFC3550 Section-5.3.1

### DISCUSSION THREAD

**Discussion by srinivasansharath** [bluenviron/mediamtx#422] [Questions] [answered]
I have incorporated RTP header extension headers in FFMPEG, but get frame corruption when the stream passes thru the rtsp-simple-server and received by a client. Just wanted to check with the knowledgeable folk here if extension headers are supported in the server. If not, can you point me in the right direction to implement it in here?

**Comment by srinivasansharath** [comment, marked as answer]
Looks like I was mistaken,, the RTP Extensions are supported, I had some bug in my code which was causing the frame corruptions..

**Reply by aler9** [reply]
RTP is handled by pion/rtp, that supports extensions.

[a comment by an automated account is not shown]
